src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
+    double areaCalculator();
+}

```

Comment:
```suggestion
public interface AreaCalculator {
    double calculateArea();
}
```
This interface is not an Area itself)
And method names must be verbs